contentcuration/contentcuration/models.py

```diff
@@ -1339,6 +1357,12 @@ def filter_view_queryset(cls, queryset, user):
             | Q(public=True)
         )
 
+    @classmethod
+    def search(self, queryset, search_term):
+        search_query = Q(title_description_search_vector=SearchQuery(value=search_term, config=POSTGRES_FTS_CONFIG, search_type="plain"))
```

Comment:
Wondering why we don't also include tag names in the search vector, rather than making a separate query here? Seems like the search vector would be an efficient lookup for any text content?

Reply:
This was done to keep the creation and updation of tsvector overhead to be as minimal as possible but now as I re-think, I feel we can do this efficiently by updating `map_tags_to_node`'s implementation to add tags to our search vector.